modules/alloy/src/main/ts/ephox/alloy/behaviour/positioning/ActivePosition.ts

```diff
@@ -5,7 +5,11 @@ import { PositioningConfig } from './PositioningTypes';
 const exhibit = (base: DomDefinitionDetail, posConfig: PositioningConfig): DomModification.DomModification =>
   DomModification.nu({
     classes: [ ],
-    styles: posConfig.useFixed() ? { } : { position: 'relative' }
+    styles: {
+      fixed: { },
+      relative: { position: 'relative' },
+      absolute: { position: 'absolute', width: '100%' }
```

Comment:
Hmm. The `width: 100%` seems a bit special-cased. What's the reason for it? Can we put an explanatory comment here about why position: absolute needs a width: 100% and other things don't.

Remember, there is supposed to be nothing about `Positioning` that makes it a tinymce sink. That's just how we use it. But Positioning could just be on something inside. In fact, if you look at some of the sketchers, you'll see that some have the option of an "internal sink". The idea there is that it would let you show the menu inside the DOM hierarchy itself, which we wanted to keep as an option if we needed it for accessibility.

Reply:
Yeah, the reason for the `width: 100%` is that absolute-positioned elements' width collapse to 0 or its contents. This makes quite a lot of dropdowns look ugly and squished, cause they do not have enough width. This can be fixed in the CSS as well but a lot more hardly.

Reply:
Added a comment [here](https://github.com/tinymce/tinymce/pull/8294/commits/d345b25bde70cbfcdf6cbe658928b7ea6dca3cc5).